packages/core/src/index.ts

```diff
@@ -125,7 +125,17 @@ const initialize = async ({
   // Only attempt to check for CodePush bundle URL in release mode. Otherwise CodePush will throw an exception.
   // https://docs.microsoft.com/en-us/appcenter/distribution/codepush/react-native#plugin-configuration-ios
   if (!__DEV__) {
-    NativeModules.EmbraceManager.checkAndSetCodePushBundleURL();
```

Comment:
with these being switched to using `await` they now have the possibility of raising an exception if the promises are rejected, probably should wrap this whole section in a try/catch to avoid crashing if any of them fail